src/firefly/java/edu/caltech/ipac/visualize/plot/Rotate.java

```diff
@@ -5,8 +5,46 @@
 
 public class Rotate
 {
-
-public static boolean debug = false;
+    /**
+     * DM-8845
+     * While working on this ticket, the algorithm is added for later references
+     *
+     * Basic rotation
+     * A basic rotation (also called elemental rotation) is a rotation about one of the axes of a
+     * Coordinate system. The following three basic rotation matrices rotate vectors by an angle θ
+     * about the x, y, or z axis, in three dimensions, using the right-hand rule—which codifies their
+     * alternating signs. (The same matrices can also represent a clockwise rotation of the axes.[nb 1])
+     *
+     * Rx(angle) = {
+     *
+     *     1            0            0
+     *     0       cos(angle)     - sin(angle)
+     *     0       sin(angle)       cos(angle)
+     *
+     *     }
+     *
+     * Ry(angle) = {
+     *
+     *      cos(angle)    0         - sin(angle)
```

Comment:
Yes, I have the correct document in RotateTest.java but Roate.java.  There is nothing documented there when I started working on this ticket.  I added the algorithm in RotateTest.java.   I also added some in Rotate.java.  I created another ticket for Rotate.java.  I will update the document and the code then.